docs/getting-started/index.md

```diff
@@ -3,52 +3,59 @@ title: Getting started
 section: getting-started
 ---
 
-Developers should have some prior knowledge of the [Shopify app ecosystem](https://shopify.dev/concepts/apps). Currently, Shopify App CLI creates apps using either [Node.js](https://nodejs.org/) or [Ruby on Rails](https://rubyonrails.org/).
+Developers should have some prior knowledge of the [Shopify app ecosystem](https://shopify.dev/concepts/apps). Shopify App CLI creates apps using either [Node.js](https://nodejs.org/) or [Ruby on Rails](https://rubyonrails.org/).
 
-## Install
+## Requirements
 
-Shopify App CLI installs using a shell script. Download and run it in your terminal with one command:
+- [Ruby](https://www.ruby-lang.org) 2.5.1+ 
+- [Node.js](https://nodejs.org) 10.0.0+
+- [curl](https://curl.haxx.se) 7.0.0+
+- A [Shopify partner account](https://partners.shopify.com/signup)
+- A [Shopify development store](https://help.shopify.com/en/partners/dashboard/development-stores#create-a-development-store) to install and test apps
+- An [ngrok](https://ngrok.com/) account (free or paid) for local development
 
-### Mac OS and Ubuntu
-```console
-eval "$(curl -sS https://raw.githubusercontent.com/Shopify/shopify-app-cli/master/install.sh)"
-```
+### Windows requirements
+
+You’ll need to install the following tools to use Shopify App CLI on Windows:
 
-### Windows
-Install [Linux Subsystem for Windows](https://docs.microsoft.com/en-us/windows/wsl/install-win10) and the [Ubuntu VM](https://www.microsoft.com/en-ca/p/ubuntu/9nblggh4msv6), then:
+- [Linux Subsystem for Windows](https://docs.microsoft.com/en-us/windows/wsl/install-win10)
+- [Ubuntu VM](https://www.microsoft.com/en-ca/p/ubuntu/9nblggh4msv6)
+
+## Install
+
+Shopify App CLI installs using a shell script. Download and run it in your terminal with one command:
 
 ```console
 eval "$(curl -sS https://raw.githubusercontent.com/Shopify/shopify-app-cli/master/install.sh)"
 ```
 
-> NOTE: Installing the Shopify App CLI requires [curl](https://curl.haxx.se/). You can to see if it's on your system by running: `curl --version`
+([View shell script source](https://raw.githubusercontent.com/Shopify/shopify-app-cli/master/install.sh))
```

Comment:
The parens look a little funny to me IRL… let's omit?

![image](https://user-images.githubusercontent.com/71433/80518646-be3abc80-8954-11ea-8800-4f244fc9e540.png)

Reply:
Sure!